pkg/pdutil/pd.go

```diff
@@ -153,15 +161,21 @@ func NewPdController(
 			}
 		}
 		processedAddrs = append(processedAddrs, addr)
-		_, failure = pdRequest(ctx, addr, clusterVersionPrefix, cli, http.MethodGet, nil)
+		versionBytes, failure = PDRequest(ctx, addr, clusterVersionPrefix, cli, http.MethodGet, nil)
 		if failure == nil {
 			break
 		}
 	}
 	if failure != nil {
-		return nil, errors.Annotatef(failure, "pd address (%s) not available, please check network", pdAddrs)
+		return nil, errors.Annotatef(berrors.ErrPDUpdateFailed, "pd address (%s) not available, please check network", pdAddrs)
 	}
 
+	version, err := semver.NewVersion(string(versionBytes))
+	if err != nil {
+		log.Warn("fail back to v0.0.0 version",
+			zap.Binary("version", versionBytes), zap.Error(err))
+		version = semver.New("0.0.0")
```

Comment:
```suggestion
		version = &semver.Version{Major: 0, Minor: 0, Patch: 0}
```

or `new(semver.Version)` also produces v0.0.0 but less explicit.